common/src/main/java/com/example/android/uamp/media/MusicService.kt

```diff
@@ -369,6 +372,17 @@ open class MusicService : MediaBrowserServiceCompat() {
                 }
             }
         }
+        override fun onPlayerError(error: ExoPlaybackException) {
+            when (error.type) {
+                ExoPlaybackException.TYPE_SOURCE -> {
+                    Toast.makeText(applicationContext,
+                        R.string.error_media_not_found,
+                        Toast.LENGTH_LONG).show()
+                    Log.e(TAG, "TYPE_SOURCE: " + error.sourceException.message)}
+                ExoPlaybackException.TYPE_RENDERER -> Log.e(TAG, "TYPE_RENDERER: " + error.rendererException.message)
```

Comment:
Lines should be wrapped at 100 chars.

Reply:
yeah, you are right! I reformatted code. Thanks ;)